docs/fides/docs/tutorial/policy.md

```diff
@@ -94,7 +94,7 @@ As global privacy laws change and businesses scale, a company's policies will ev
 
 ## Check Your Progress
 
-After making the above changes and the changes in the previous two steps, your app should resemble the state of the [`ethyca/fidesdemo` repository](https://github.com/ethyca/fidesdemo) at the [`fidesctl-manifests`](https://github.com/ethyca/fidesdemo/releases/tag/fidesctl-manifests) tag.
+After making the above changes and the changes in the previous two steps, your app should resemble the state of the [`ethyca/fidesdemo` repository](https://github.com/ethyca/fidesdemo) at the [`fidesctl-manifests` tag](https://github.com/ethyca/fidesdemo/releases/).
```

Comment:
@PSalant726 looking for your input here, i've updated the 2 places where there were broken links from the demo tutorial with this, i think linking to "releases" on fidesdemo is sufficient?

Reply:
@NevilleS any thoughts or recommendations? Looks like this tag was removed from `fidesdemo` as well.

Reply:
No I'd like these tags to be there, it's just a bug that `fidesdemo` doesn't have them

Reply:
got it, i'll change back, thanks Neville!